### DISCUSSION TITLE
New Resource Status

### DISCUSSION THREAD

**Discussion by Midorip** [centreon-archive/centreon-archived#9607] [General]
Hello everyone,

I find very difficult to use the new Resources Status page, we are still using the Status Detailed page, specially for the graph or other actions
Since those pages are deprecated, it will probably be decomissionned on the next major version.
Do you have any information regarding this ? 

Thanks

**Comment by lpinsivy** [comment]
Hi @Midorip,

With Centreon 21.04 we made severals changes on this page. 
you can find the list of enhancements [here](https://docs.centreon.com/21.04/en/releases/centreon-core.html#enhancements).

Regards
